ultra/ultra/baselines/agent_spec.py

```diff
@@ -82,8 +84,6 @@ def __new__(
                 social_capacity=10,
             )
             adapter = BaselineAdapter(social_vehicle_params=social_vehicle_params)
-            policy_dir = "/".join(inspect.getfile(policy_class).split("/")[:-1])
```

Comment:
why did u remove these? policy_params should be read from the saved version 
if you don't read the policy_params from the saved file, it could lose its original values from when the experiment was started.  here is the problem scenario:
 1- start experiment 1 with param x = 100
 2- start experiment 2 with param x = 200
 3- experiment 1 starts evaluation and creates AgentSpec with x = 200 instead of x = 100

so for avoiding this situation I saved the params from the beginning of each experiment and load them once evaluation of that experiment need them

Reply:
I am too sure what it means by "policy params should be read from the saved version". Currently I have the read the config file and extract those policy params here:

```         
            else:
                 policy_dir = "/".join(inspect.getfile(policy_class).split("/")[:-1])
                 policy_params = load_yaml(f"{policy_dir}/params.yaml")
                 social_vehicle_params = dict(	           
```
Instead of extracting the policy params after line 86, I moved it up (still in the else clause) so I can extract the social vehicle params at the same time and get the rest of the policy params.

Reply:
ok I checked your agent_spec and those lines were not removed.